synth.py

```diff
@@ -86,32 +69,16 @@ def generate_data_api(gapic, license):
     "google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/GrpcBigtableCallableFactory.java",
   ]
 
-  package_name = f'com.google.cloud.bigtable.v2'
-  java.fix_proto_headers(library / f'proto-google-cloud-bigtable-v2')
-  # java.fix_grpc_headers(library / f'grpc-google-cloud-bigtable-v2', package_name)
-
-  ##todo replace with fix_grpc_headers
-  s.replace(
-      library / f'grpc-google-cloud-bigtable-v2/src/**/*.java',
-      f"package com.google.bigtable.v2;",
-      f"{license}package com.google.bigtable.v2;"
-  )
-  s.copy(library / f'gapic-google-cloud-bigtable-v2', 'google-cloud-bigtable/', excludes=excludes)
-  s.copy(library / f'grpc-google-cloud-bigtable-v2/src', f'grpc-google-cloud-bigtable-v2/src')
-  s.copy(library / f'proto-google-cloud-bigtable-v2/src', f'proto-google-cloud-bigtable-v2/src')
-
   make_internal_only(internal_only)
 
-  java.format_code(f'./grpc-google-cloud-bigtable-v2/src')
-  java.format_code(f'./proto-google-cloud-bigtable-v2/src')
-
-
-def generate_admin_api(gapic, license):
-  library = gapic.java_library(
-      service='bigtable-admin',
-      version='v2',
-      config_path='/google/bigtable/admin/artman_bigtableadmin.yaml',
-      artman_output_name='')
+def generate_admin_api(gapic):
+  java.gapic_library(
+        service='bigtable-admin',
+        version='v2',
+        config_pattern='/google/bigtable/admin/artman_bigtableadmin.yaml',
+        package_pattern='com.google.bigtable.v2',
+        gapic=gapic,
+  )
```

Comment:
I don't think this will put these files in the right place. java-bigtable is a single artifact that contains clients for both admin & data. Which is why all of the explicit copies were added